cardano-cli/src/Cardano/CLI/Commands/Legacy.hs

```diff
@@ -68,32 +68,32 @@ import           Data.Time.Clock
 -- | All the CLI subcommands under \"shelley\".
 --
 data LegacyCommand
-  = AddressCmds       AddressCmds
-  | StakeAddressCmds  StakeAddressCmds
-  | KeyCmds           KeyCmds
-  | TransactionCmds   TransactionCmds
-  | NodeCmds          NodeCmds
-  | PoolCmds          PoolCmds
-  | QueryCmds         QueryCmds
-  | GovernanceCmds    GovernanceCmds
-  | GenesisCmds       GenesisCmds
-  | TextViewCmds      TextViewCmds
+  = LegacyAddressCmds       LegacyAddressCmds
```

Comment:
Suggestion: prefix constructors with `L` instead of `Legacy` for better readability. The meaning still would be obvious enough I think.